git-date/src/parse.rs

```diff
@@ -55,8 +58,11 @@ pub fn parse(input: &str) -> Result<Time, Error> {
         } else if let Some(val) = parse_raw(input) {
             // Format::Raw
             Ok(val)
-        } else if let Some(val) = relative::parse(input) {
-            Ok(Time::new(val.unix_timestamp() as u32, val.offset().whole_seconds()))
+        } else if let Some(val) = relative::parse(input, now.ok_or(Error::MissingCurrentTime)?) {
```

Comment:
> This would cause unnecessary errors if no time is given but it's not a relative date either.

See https://github.com/Byron/gitoxide/pull/498/commits/1eac4de99de250e770b48054fd5bc6b806418e4d#diff-fe1f4921b662e3ab738c66e74ec7da02aa5ffde9cc94fe9c1a17e90326f54813R74 no error if passing None